jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1028,6 +1031,21 @@ public void addEndpointSsrc(@NotNull AbstractEndpoint endpoint, long ssrc)
         }
     }
 
+    public List<Long> getAllRemoteAudioSsrcs(String localEndpointId) {
+        List<Long> ssrcs = new ArrayList<>();
+        for (AbstractEndpoint endpoint : getEndpoints()) {
```

Comment:
If you use getLocalEndpoints instead you can remove the `instanceof` check

Reply:
Used getLocalEndpoints and removed type check